gtsam/symbolic/tests/testSymbolicFactorGraph.cpp

```diff
@@ -124,14 +124,40 @@ TEST(SymbolicFactorGraph, eliminatePartialMultifrontal) {
 }
 
 /* ************************************************************************* */
-TEST(SymbolicFactorGraph, marginalMultifrontalBayesNet) {
-  auto expectedBayesNet =
-      SymbolicBayesNet(SymbolicConditional(0, 1, 2))(SymbolicConditional(
-          1, 2, 3))(SymbolicConditional(2, 3))(SymbolicConditional(3));
+TEST(SymbolicFactorGraph, MarginalMultifrontalBayesNetOrdering) {
+  auto expectedBayesNet = SymbolicBayesNet({0, 1, 2})({1, 2, 3})({2, 3})({3});
 
-  SymbolicBayesNet actual1 =
+  SymbolicBayesNet actual =
       *simpleTestGraph2.marginalMultifrontalBayesNet(Ordering{0, 1, 2, 3});
-  EXPECT(assert_equal(expectedBayesNet, actual1));
+  EXPECT(assert_equal(expectedBayesNet, actual));
+}
+
+TEST(SymbolicFactorGraph, MarginalMultifrontalBayesNetKeyVector) {
+  auto expectedBayesNet = SymbolicBayesNet({0, 1, 2})({2, 1, 3})({1, 3})({3});
+
+  SymbolicBayesNet actual =
+      *simpleTestGraph2.marginalMultifrontalBayesNet(KeyVector{0, 1, 2, 3});
```

Comment:
Can you add comment or rename that this is testing COLAMD ordering?  When reviewing I initially read this file and got confused until I read EliminateableFactorGraph and remembered that KeyVector is treated unordered.